src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -235,7 +235,7 @@ void M()
             };
 
             await VerifyNoMultiLineTokens(testLspServer, results).ConfigureAwait(false);
-            Assert.Equal(expectedResults.Data, results);
+            AssertEx.Equal(expectedResults.Data, results);
```

Comment:
@CyrusNajmabadi out of curiosity what's the difference between `Assert` and `AssertEx` here?

Reply:
@allisonchou it prints a clearer error message (with a diff) about what is different. Assert just says they're different. AssertEx shows where that difference is and what it is. Very useful for these large arrays of numbers.